processor/ratelimitprocessor/gubernator.go

```diff
@@ -102,17 +115,32 @@ func (r *gubernatorRateLimiter) RateLimit(ctx context.Context, hits int) error {
 	})
 	if err != nil {
 		r.set.Logger.Error("error executing gubernator rate limit request", zap.Error(err))
+		r.telemetryBuilder.RatelimitRequests.Add(ctx, 1, metric.WithAttributeSet(attribute.NewSet(
+			telemetry.WithProjectID(projectID),
+			telemetry.WithLimitReason("request_error"),
+			telemetry.WithDecision("rejected"),
```

Comment:
@vigneshshanmugam I think `rejected` is the right one here but looking at apm code https://github.com/elastic/apm-ingest-service/blob/main/internal/ratelimit/ratelimit.go#L314 it's using `accepted` as the decision here.

Reply:
To clarify, We should accept requests instead of rejecting the with the error when ratelimiter component was down for some reason, So MIS is doing the right thing here.